app/util/autoversioning.py

```diff
@@ -170,3 +175,12 @@ def _execute_local_git_command(*args):
         cwd=os.path.dirname(__file__),
     )
     return command_output.decode()
+
+
+if __name__ == '__main__':
+    """
+    Print version string.
+    """
+    version = get_version()
+    write_package_version_file(version)
+    print(version)
```

Comment:
You might want to call `restore_original_package_version_file` after printing the version so it doesn't leave the git repo dirty. It doesn't matter too much but is a nice-to-have. (See setup.py for how it's being called there.)